### DISCUSSION TITLE
latest version

### DISCUSSION THREAD

**Discussion by fcrisciani** [hassio-addons/addon-bitwarden#96] [General]
Hi,
is there still a technical issue in following the upstream bitwarden repo and get the addon on the latest release?

**Comment by frenck** [comment]
No?

**Reply by fcrisciani** [reply]
I miss the fact that this repo is based on github.com/dani-garcia/bitwarden_rs that is different from github.com/bitwarden/server

**Reply by frenck** [reply]
Correct.

**Reply by fcrisciani** [reply]
looks like this question should go one layer above :) Thanks for the fast reply!

**Reply by surapanenivenu** [reply]
the RS version web vault is now 2.19 and supports send API

**Reply by fcrisciani** [reply]
yeah but we need to wait for dani-garcia to cut a new release to see it here

**Comment by damanti-me** [comment]
Bitwarden_RS just released 1.20. 👍

**Reply by frenck** [reply]
If you actually looked at the repository before commenting; you would have noticed that its already in dev.

**Reply by damanti-me** [reply]
nice!
